### PR TITLE
Mark nullable fields in PHPDoc

### PR DESCRIPTION
**Pull request by ob-stripe**
r? @remi-stripe 
cc @stripe/api-libraries 

- Mark nullable fields in PHPDoc
- Use fully qualified names everywhere for `\Stripe\Collection`, `\Stripe\StripeObject`, etc.

### CODE REVIEW

File: lib/Radar/ValueListItem.php [hunk 1]
```diff
@@ -17,10 +17,10 @@
  */
 class ValueListItem extends \Stripe\ApiResource
 {
-    const OBJECT_NAME = "radar.value_list_item";
+    const OBJECT_NAME = 'radar.value_list_item';
 
-    use \Stripe\ApiOperations\All;
     use \Stripe\ApiOperations\Create;
     use \Stripe\ApiOperations\Delete;
+    use \Stripe\ApiOperations\All;
```
Comment (remi-stripe) [inline lib/Radar/ValueListItem.php]: Nice catch but this needs a test

Comment (ob-stripe) [reply lib/Radar/ValueListItem.php]: Actually that was just the codegen tool moving the trait, not adding it. I've reverted the change.

**Comment by ob-stripe** [comment]
Released as 7.8.0.

**Comment by ruudk** [comment]
Thanks so much 💙

**Pull request merged**